lib/model/query/entities.js

```diff
@@ -209,7 +216,8 @@ const _get = (includeSource) => {
     LEFT JOIN actors current_version_actors ON current_version_actors.id=entity_defs."creatorId"
   `}
   ${!includeSource ? sql`` : sql`
-    LEFT JOIN submission_defs ON submission_defs.id = entity_defs."submissionDefId"
+    LEFT JOIN entity_def_sources ON entity_defs."sourceId" = entity_def_sources."id"
+    LEFT JOIN submission_defs ON submission_defs.id = entity_def_sources."submissionDefId"
```

Comment:
this is clunky - i took out the part below that builds the source response but left this part of the query in.